docs/advanced/document-store/index.md

```diff
@@ -161,6 +161,7 @@ DocArray supports multiple storage backends with different search features. The
 | [`ElasticSearch`](./elasticsearch.md) | `DocumentArray(storage='elasticsearch')` | ✅             | ✅                      | ✅             |
 | [`Redis`](./redis.md)                | `DocumentArray(storage='elasticsearch')` | ✅             | ✅                      | ✅             |
 
+Detailed information and recommendations on choosing the right storage backend can be found in our [**One Million Scale Benchmark**](./benchmark#conclusion)
```

Comment:
I would say something like "choosing the right backends depends on your data scale and the backend performance. Check our One Million Scale Benchmark for more details

Reply:
i've updated the text. Please check again @alaeddine-13